### ISSUE TITLE
Laravel 4.1 support broken since "Forget to pull"

### ISSUE THREAD

**Issue by pierrerigal** [ipunkt/laravel-analytics#13]
Hello,

Pull is not supported in Laravel 4.1 so Laravel 4.1 support is broken.

In TrackingBag.php :

You wrote :  

````php
public function get()
	{
		if (Session::has($this->sessionIdentifier)) {
			return Session::pull($this->sessionIdentifier);
		}

		return [];
	}
````
instead of : 

````php
public function get()
	{
		if (Session::has($this->sessionIdentifier)) {
			$trackings = Session::get($this->sessionIdentifier);

			//  forget the session store data
			Session::forget($this->sessionIdentifier);

			return $trackings;
		}

		return [];
	}
`````

**Comment by pierrerigal** [comment]
Moreover, it will fail only when in production where analytics is active. In local, when using NoAnalytics, there is no error :)

Big surprise for me when deploying this morning ! (in staging, not in prod)

**Comment by rokde** [comment]
Thank you for your Bug report. I have fixed that Issue with Release 1.1.1 (https://github.com/ipunkt/laravel-analytics/tree/1.1.1)

**Comment by pierrerigal** [comment]
thx !
